avocado/utils/vmimage.py

```diff
@@ -388,7 +388,8 @@ def list_providers():
     return set(_ for _ in itervalues(globals())
                if (_ != ImageProviderBase and
```

Comment:
So, maybe now we can remove the check for `!= ImageProviderBase` and keep the conditions little bit simpler, right?

Reply:
True!